lib/engine.py

```diff
@@ -707,8 +707,8 @@ def filename_from_url(url):
 
 def gen_from_source(source):
     """Returns a generator from a source tuple.
-    
-    Source tuples are of the form (callable, args) where callable(*args)
+
+    Source tuples are of the form (callable, args) where callable(\*args)
```

Comment:
I don't see an option to merge here for me. Perhaps I am not added to weecology group, so maybe thats why.